lib/src/install/baseline.rs

```diff
@@ -113,11 +119,11 @@ fn sgdisk_partition(
 fn mkfs<'a>(
     dev: &str,
     fs: Filesystem,
-    label: Option<&'_ str>,
+    label: &str,
     opts: impl IntoIterator<Item = &'a str>,
 ) -> Result<uuid::Uuid> {
     let u = uuid::Uuid::new_v4();
-    let mut t = Task::new("Creating filesystem", format!("mkfs.{fs}"));
+    let mut t = Task::new("Creating {label} filesystem ({fs}", format!("mkfs.{fs}"));
```

Comment:
description needs wrapped in `format!` here, also missing closing ')' in description str

Reply:
Thanks!  I had apparently forgotten to `git commit --amend`